sitecore-xp0-sxa-horizon/docker/.env

```diff
@@ -0,0 +1,33 @@
+COMPOSE_PROJECT_NAME=konabos-sxa
```

Comment:
Can you have a unique compose project name, i.e. konabos-sxa-horizon
This helps when building multiple images and avoids clashes